### PR TITLE
Re-introduce rust-analyzer-contributors

### PR DESCRIPTION
**Pull request by Veykril**
We kind of have a use for this again now

@rust-lang/rust-analyzer

### CODE REVIEW

**Comment by Veykril** [comment]
cc @alibektas, @davidbarsky, @Nadrieril

File: teams/rust-analyzer.toml [hunk 1]
```diff
@@ -7,7 +7,6 @@ members = [
     "lnicola",
     "Veykril",
     "flodiebold",
-    "bjorn3",
```
Comment (rylev) [inline teams/rust-analyzer.toml:10]: Should this be removed to `alumni` or was the inclusion a mistake?

**Review by rylev — CHANGES_REQUESTED** [review]

File: teams/rust-analyzer.toml (hunk 1 above)
Comment (Veykril) [reply teams/rust-analyzer.toml:10]: That was intentional, `bjorn3` was only in the team for issue triaging permissions when we removed the contributors team. I've moved them to the contributors back now with this.

**Comment by Nadrieril** [comment]
I notice that my `people/Nadrieril.toml` file doesn't have a `zulip-id`. If this is important for r-a, my id is 245339

**Comment by Veykril** [comment]
I don't think it is

File: teams/rust-analyzer.toml (hunk 1 above)
Comment (bjorn3) [reply teams/rust-analyzer.toml:10]: LGTM

**Pull request merged**